UnityProject/Assets/Scripts/Objects/Engineering/APCPoweredDevice.cs

```diff
@@ -114,8 +120,16 @@ void Start()
 			}
 		}
 
+		public void damage(int x)
```

Comment:
Use PascalCase. Could use the same name as the method it is delegating functions to, ``ApplyDamage``, also don't use ``x`` for parameter names, be more descriptive. If this method is used only in one place, you could simply expose the reference to integrity component using a property
```cs
public Integrity Integrity => integrity;
```